.github/workflows/sync-submodules.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  schedule:
+    # Run at 11:11 (UTC) every Monday
+    - cron:  '11 11 * * 1'
+  # Allow running manually, from the Actions tab or through the GitHub HTTP API
+  workflow_dispatch:
+
+name: Sync Submodules
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Git submodule update
+      run: |
+        git submodule update --init --recursive --remote
```

Comment:
It would be better to only update the submodules that we want over updating everything, this may break stuff.

Reply:
you are correct, lets just wait for the stable release to merge this.